docs/nx-cloud/reference/nx-cloud-cli.md

```diff
@@ -5,6 +5,12 @@
 At the beginning of your main job, invoke `npx nx-cloud start-ci-run`. This tells Nx Cloud that the following series of
 command correspond to the same CI run.
 
+{% callout type="warning" title="Do not run start-ci-run locally" %}
+`nx-cloud start-ci-run` is designed to run in CI. If you run the command locally, it can cause your local Nx repo think it's apart of a CI run. This can cause strange behavior like Nx commands timing out or throwing unexpected errors.
```

Comment:
```suggestion
`nx-cloud start-ci-run` is designed to run in CI. If you run the command locally, it can cause your local Nx repo to think it's a part of a CI run. This can cause strange behavior like Nx commands timing out or throwing unexpected errors.
```